conan/tools/files/files.py

```diff
@@ -442,6 +442,7 @@ def untargz(filename, destination=".", pattern=None, strip_root=False, extract_f
         if not pattern and not strip_root:
             tarredgzippedFile.extractall(destination)
         else:
+            using_long_path_prefix = destination.startswith("\\\\?\\")
```

Comment:
```suggestion
            using_long_path_prefix = destination.startswith(r"\\?\")
```

Reply:
I tried that at first, but it raised a syntax error:

```
SyntaxError: unterminated string literal (detected at line 445); perhaps you escaped the end quote?
```

So it should be `r"\\?\\"` instead.  That's why I decided to use this one 🤷‍♂️ 

```suggestion
            using_long_path_prefix = destination.startswith(r"\\?\\")
```
Do you prefer this one?

Reply:
I don't get it. If using the ``r``, I thought it was possible to use the exact literal string that @AbrilRBS proposed

Reply:
```bash
conan/tools/scm/git.py:5: in <module>
    from conan.tools.files import chdir, update_conandata
conan/tools/files/__init__.py:1: in <module>
    from conan.tools.files.files import load, save, mkdir, rmdir, rm, ftp_download, download, get, \
E     File "/Users/conan/conan/tools/files/files.py", line 445
E       using_long_path_prefix = destination.startswith(r"\\?\")
E                                                       ^
E   SyntaxError: unterminated string literal (detected at line 445); perhaps you escaped the end quote?
```

More:
* https://stackoverflow.com/questions/647769/why-cant-pythons-raw-string-literals-end-with-a-single-backslash#:~:text=String%20quotes%20can%20be%20escaped,an%20odd%20number%20of%20backslashes